### PR TITLE
Article: Front End Tribute Page

### PR DESCRIPTION
**Pull request by theoneinskane**
Issue #539 
created a wiki page for the front end project Build a Tribute Page.
Added in what I consider to be tips that I found useful when I first
started designing simple web pages.

### CODE REVIEW

**Comment by Rafase282** [comment]
`Tribute-Page.md` would do fine. We could come up with a shorter key terms for these kind of articles. Lets talk about it on the chat.

File: Project-Tribute-Page.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
+
+**List the features required by the user stories and any other rules you must fulfill**
```
Comment (abhisekp) [inline Project-Tribute-Page.md]: Use "heading 2"
```md
## List ...
```

Comment (abhisekp) [inline Project-Tribute-Page.md]: Use heading 3 here and for rest, use heading 2

File: Project-Tribute-Page.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
+
+##List the features required by the user stories and any other rules you must fulfill
+###Just write them down on a piece of paper. Having them on paper and not in a file will make it quicker to refer to them during the design process.
```
Comment (abhisekp) [inline Project-Tribute-Page.md]: use normal text

File: Project-Tribute-Page.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
+
+##List the features required by the user stories and any other rules you must fulfill
+###Just write them down on a piece of paper. Having them on paper and not in a file will make it quicker to refer to them during the design process.
+
+##Layout your page on paper
+###It might sound a bit silly but this is a great way to start visualizing what the page should look like. Elements of the page you should start with could be a title, images, text and links.
```
Comment (abhisekp) [inline Project-Tribute-Page.md]: use normal text here

File: Project-Tribute-Page.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+##So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
```
Comment (abhisekp) [inline Project-Tribute-Page.md]: Normal text here

File: Project-Tribute-Page.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+##So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
+
+##List the features required by the user stories and any other rules you must fulfill
```
Comment (abhisekp) [inline Project-Tribute-Page.md]: use short headings. And then use a subtitle.

E.g. 
```md
## List Features
**List the features required by the user stories and any other rules you must fulfill**
```

which would appear as
> ## List Features
**List the features required by the user stories and any other rules you must fulfill**

File: Project-Tribute-Page.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+##So this is pretty much the first full webpage you might have ever created. Some tips to help get you with this:
+
+##List the features required by the user stories and any other rules you must fulfill
+Just write them down on a piece of paper. Having them on paper and not in a file will make it quicker to refer to them during the design process.
+
+##Layout your page on paper
+It might sound a bit silly but this is a great way to start visualizing what the page should look like. Elements of the page you should start with could be a title, images, text and links.
```
Comment (theflametrooper) [inline Project-Tribute-Page.md]: Maybe explain the word "wireframe"

**Comment by alayek** [comment]
Hey @theoneinskane, are you working on this? If you have any questions or need any help; feel free to ask!

**Comment by theoneinskane** [comment]
@alayek I'm away at my wedding right now. Will get to T when I'm back Wednesday!

**Comment by abhisekp** [comment]
@theoneinskane you don't need to close the pull request. Please feel free to re-open this and continue with this PR.

**Comment by alayek** [comment]
@theoneinskane first of all, if I read that right, congrats man!! Also, as @abhisekp said, you didn't need to close this PR. 

I am re-opening this PR for now. You can continue adding more commits on top of it.

**Comment by Rafase282** [comment]
Made changes requested https://github.com/theoneinskane/wiki/pull/1

**Pull request merged**